test/mpi/.gitignore

```diff
@@ -1519,3 +1519,176 @@
+/maint/f77tof90
+/binary_test
+/dtpools/stamp-h1
+/dtpools/confdb/ar-lib
+/dtpools/confdb/compile
+/dtpools/confdb/missing
+/dtpools/confdb/depcomp
+/dtpools/confdb/install-sh
+/dtpools/autom4te.cache/requests
+/spawn/spawn_rootargs
+/confdb/ar-lib
+/confdb/compile
+/confdb/missing
+/confdb/depcomp
+/confdb/install-sh
+/datatype/dataalign2
+/datatype/type_large
+/datatype/pack_external
+/include/stamp-h1
+/rma/fetch_and_op_long_accop_hwacc
+/rma/atomic_get_long_double_int
+/rma/window_attach
+/rma/atomic_get_2int
+/rma/fetch_and_op_double_accop_hwacc
+/rma/fetch_and_op_char_hwacc
+/rma/fetch_and_op_long_hwacc
+/rma/win_dynamic_rma_flush_get_collattach
+/rma/atomic_get_double_int
+/rma/win_dynamic_rma_flush_get
+/rma/atomic_get_short_int
+/rma/win_dynamic_acc_collattach
+/rma/win_shared_query_null
+/rma/fetch_and_op_long_double_accop_hwacc
+/rma/fetch_and_op_long_double_hwacc
+/rma/fetch_and_op_char_accop_hwacc
+/rma/fetch_and_op_int_accop_hwacc
+/rma/fetch_and_op_short_hwacc
+/rma/fetch_and_op_double_hwacc
+/rma/aint_collattach
+/rma/fetch_and_op_int_hwacc
+/rma/atomic_get_long_int
+/rma/window_attach_collattach
+/rma/fetch_and_op_short_accop_hwacc
+/rma/atomic_get_float_int
+/impls/mpich/mpi_t/collparmt
+/impls/mpich/mpi_t/recvq_events
+/impls/mpich/comm/comm_info_hint
+/impls/mpich/comm/comm_dup
+/threads/pt2pt/ibsend
+/threads/part/parrived_wait
+/threads/comm/comm_create_group_threads2
+/threads/comm/idup_deadlock
+/threads/comm/idup_comm_gen
+/threads/init/mult_session
+/threads/rma/multirma
+/threads/rma/multiget
+/autom4te.cache/requests
+/f90/io/i_setviewcurf90
+/f90/io/gen-src-stamp
+/f90/pt2pt/isendf90
+/f90/pt2pt/rsendf90
+/f90/pt2pt/issendf90
+/f90/pt2pt/prsendf90
+/f90/pt2pt/mprobef90
+/f90/pt2pt/pssendf90
+/f90/pt2pt/ssendf90
+/f90/pt2pt/irsendf90
+/f90/pt2pt/sendrecvf90
+/f90/pt2pt/sendrecvreplf90
+/f90/pt2pt/psendf90
+/f90/pt2pt/sendf90
+/f90/info/infotest2f90
+/f90/info/infotestf90
+/f90/info/infocrenvf90
+/f90/info/infogetstrf90
+/f90/comm/commnamef90
+/f90/comm/commerrf90
+/f90/init/baseenvf90
+/f90/profile/profile1f90
+/f90/topo/dgraph_wgtf90
+/f90/topo/cartcrf90
+/f90/topo/dgraph_unwgtf90
+/f90/spawn/spawnmultf03
+/f90/spawn/spawnargvf03
+/f90/datatype/bottom
+/f90/rma/aintf90
+/f90/coll/nonblocking_inpf90
+/f90/coll/vw_inplacef90
+/f90/coll/allredopttf90
+/f90/coll/exscanf90
+/f90/coll/redscatf90
+/f90/coll/reducelocalf90
+/f90/coll/allredint8f90
+/f90/coll/split_typef90
+/f90/coll/alltoallwf90
+/f90/coll/nonblockingf90
+/f90/coll/inplacef90
+/f90/coll/red_scat_blockf90
+/f90/coll/alltoallvf90
+/f90/coll/uallreducef90
+/coll/bcast_comm_world_only
+/coll/p_scatterv
+/coll/p_barrier
+/coll/p_neighb_alltoallv
+/coll/p_neighb_allgatherv
+/coll/p_scan
+/coll/bcast
+/coll/p_neighb_alltoall
+/coll/p_exscan
+/coll/p_gatherv
+/coll/p_scatter
+/coll/coll_large
+/coll/p_redscat
+/coll/p_neighb_alltoallw
+/coll/p_red_scat_block
+/coll/gatherv
+/coll/p_neighb_allgather
+/coll/p_gather
+/coll/uoplong_large
+/coll/uop_equal
```

Comment:
It would be nice if we could keep these grouped together (alphabetically sorted) with tests from the same directory rather than tacked on at the end of the file.

Reply:
I advised him to keep it separate to ease the review. @travisk1154 could you sort them with a 2nd commit?

Reply:
Yea sure thing. Should there also be a separate commit for deletions from .gitignore?

Reply:
You can delete the entries in the first commit. It doesn't affect the review. In addition, if you *insert* the new entries rather than reorder the existing ones, you can do all of them in one commit as well.